src/hle.c

```diff
@@ -170,6 +159,7 @@ static void rsp_break(struct hle_t* hle, unsigned int setbits)
 static void forward_gfx_task(struct hle_t* hle)
 {
     HleProcessDlistList(hle->user_defined);
+    rsp_break(hle, SP_STATUS_TASKDONE);
```

Comment:
I guess a future change would be to let the video plugin decide whether the task is done or not. But this would only happen if we change the InitiateGFX API in order to give access to SP_STATUS_REG.
This would allow rsp task yielding.

Reply:
This is indeed a possibility.
My primary motivation for these changes was indeed to allow me to implement the gfx ucode of WDC inside the rsp-hle plugin (have a old WIP branch on my laptop). But at the time (~2015) there were still core issues which made my progress stops... Still if someone is interested I can push what I have in my repo to help a new take on this.

Reply:
You can find my old stuff here :
https://github.com/bsmiles32/mupen64plus-rsp-hle/commit/804604adf951cfa0d4c4a81c03e7d7ab9cb8cd45
It doesn't work but was WIP as I were still REing the ucode.

cc: @olivieryuyu